enterprise/backend/test/metabase_enterprise/data_editing/api_test.clj

```diff
@@ -703,3 +703,106 @@
+                {create-action "table.row/create"
+                 update-action "table.row/update"
+                 delete-action "table.row/delete"}
+                (u/index-by :kind crud-actions)]
+            (is (= 200 status))
+            (testing "table actions have neg ids"
+              (is (every? neg? (map :id crud-actions))))
+            (testing "one action for each crud op"
+              (is (= {"table.row/create" 1
+                      "table.row/update" 1
+                      "table.row/delete" 1}
+                     (frequencies (map :kind crud-actions)))))
+            (mt/with-temp [:model/Dashboard dash {}]
+              (let [dashboard-url (str "dashboard/" (:id dash))
+                    card-counter (atom 0)
+                    card-input (fn [action]
+                                 {:id (swap! card-counter dec)
+                                  :size_x 1
+                                  :size_y 1
+                                  :row 0
+                                  :col 0
+                                  :action_id (:id action)})
+                    {:keys [dashcards]}
+                    (mt/user-http-request
+                     :crowberto
+                     :put
+                     dashboard-url
+                     {:dashcards [(card-input create-action)
+                                  (card-input update-action)
+                                  (card-input delete-action)]})
+
+                    exec-url #(str dashboard-url "/dashcard/" (:id %) "/execute")
+
+                    prefill-values
+                    #(mt/user-http-request
+                      :crowberto
+                      :get
+                      (exec-url %1)
+                      :parameters (json/encode %2))
+
+                    execute!
+                    #(mt/user-http-request
+                      :crowberto
+                      :post
+                      (exec-url %1)
+                      {:parameters %2})
+
+                    [create-card
+                     update-card
+                     delete-card]
+                    dashcards]
+
+                (testing "create"
+                  (is (= {} (prefill-values create-card {})))
+                  (execute! create-card
+                            {:text      "hello, world!"
+                             :int       42
+                             :timestamp "2025-05-12 14:32:16"
+                             :date      "2025-05-12"})
+                  (execute! create-card
+                            {:text      "seeya, world!"})
+                  (is (= [[1 "hello, world!" 42 "2025-05-12T14:32:16Z" "2025-05-12T00:00:00Z"]
+                          [2 "seeya, world!" nil nil                   nil]]
+                         (->> (table-rows @test-table)
+                              (sort-by first)))))
+
+                (testing "update"
+                  (is (= {} (prefill-values update-card {})))
```

Comment:
Why do we need this vacuous seeming prefill?

Reply:
Made a note that its fine for now for it simply not to crash for cases with parameters and without